src/integrations/web.msg.post.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+function image_tp_web($imageUrl, $imageName, $password, $login, $site) {
```

Comment:
Should this be `image_to_web` or is `tp` some other shortening?

Reply:
Shouls be to web. Fixed this mistake on next commit.